ecommerce/static/js/views/course_seat_form_fields/course_seat_form_field_view.js

```diff
@@ -38,7 +38,13 @@ define([
                 'input[name=id_verification_required]': {
                     observe: 'id_verification_required',
                     onSet: 'cleanIdVerificationRequired'
-                }
+                },
+		'input[name=allowed_bin]':{
+		  observe: 'allowed_bin',
+		  setOptions: {
+			validate: true
+		  }
+		}
```

Comment:
is this part of the extend?

Reply:
What do you mean with part of the extend?

Reply:
Part of the `Backbone.View.extend` 

If it is, can you indent the code block so it's clearer